src/math/fir_hifi5.c

```diff
@@ -198,21 +188,14 @@ void fir_32x16_2x(struct fir_state_32x16 *fir, ae_int32 x0, ae_int32 x1,
 
 	/* Write samples to delay */
 	AE_S32_L_XC(x0, fir->rwp, -sizeof(int32_t));
-	dp = (ae_f32x2 *)fir->rwp;
+	dp = (ae_int32x2 *)fir->rwp;
```

Comment:
No need to change, I was trying ae_int32x4 pointer but it can't be used here due to data align issue.